integration-tests/tsconfig.json

```
@@ -0,0 +1,12 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "ES5",
```

I suggest to use ES modules everywhere. Just to avoid problems with the imports.

```suggestion
        "module": "ESNext",
        "target": "ESNext",
```